lib/connection/connection_config.js

```diff
@@ -484,6 +485,13 @@ function ConnectionConfig(options, validateCredentials, qaMode, clientInfo)
     }
   }
 
+  let disableQueryContextCache = false;
+  if (Util.exists(options.disableQueryContextCache)) {
+    Errors.checkArgumentValid(Util.isBoolean(options.disableQueryContextCache),
+      ErrorCodes.ERR_CONN_CREATE_INVALID_DISABLED_QUERY_CONTEXT_CACHE);
+      disableQueryContextCache = options.disableQueryContextCache;
```

Comment:
this line seems to start on wrong place, should be on the same level as `Errors.checkArgumentValid`